src/utils.js

```diff
@@ -134,4 +140,29 @@ const validateScript = (extension, file) => {
     return true;
   }
 };
-exports.validateScript = validateScript;
\ No newline at end of file
+exports.validateScript = validateScript;
+
+// read and parse the JSON file
+const importJson = (filename) => {
+  let content;
+  try {
+    content = fs.readFileSync(filename, "utf8");
+    return JSON.parse(content);
+  } catch (error) {
+    throw new Error('Invalid JSON import!');
+  }
+}
+exports.importJson = importJson;
+
+// checks whether the given argument is referring to an JSON object
+// if so returns the corresponding object, else returns null
+const parseJson = (argument, constants) => {
```

Comment:
function name and purpose does not match. argument names and purpose do not match.